model-optimizer/extensions/ops/random_uniform.py

```diff
@@ -51,6 +52,8 @@ def infer(node: Node):
         node.in_node(1)['correct_data_type'] = True
         node.in_node(2)['correct_data_type'] = True
 
+        PermuteInputs().set_input_permutation(node.in_node(0), node, 'output:0', 'shape')
```

Comment:
What about ONNX RandomUniform?

Reply:
Permutation will be done only if needed, it is standard way to permute. ONNX shouldn't be affected.